<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceViewer2DWidget</class>
 <widget class="QWidget" name="SliceViewer2DWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>886</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>ImageSlice2DWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Slice:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="firstSlice_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>First Slice</string>
       </property>
       <property name="text">
        <string>|&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="previousSlice_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Previous Slice</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextSlice_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Next Slice</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lastSlice_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Last Slice</string>
       </property>
       <property name="text">
        <string>&gt;|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slideSelector_slider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Scroll through the slices</string>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="transverse_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Transverse Orientation</string>
       </property>
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sagittal_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Sagittal Orientation</string>
       </property>
       <property name="text">
        <string>|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="coronal_button">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Coronal Orientation</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QVTKOpenGLStereoWidget" name="Frame2D">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="dataSelect_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="dataIndex_layout">
        <item>
         <widget class="QLabel" name="dataIndex_lbl">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Series:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="beginningSeries_button">
          <property name="toolTip">
           <string>First Series</string>
          </property>
          <property name="text">
           <string>|&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="backwardSeries_button">
          <property name="toolTip">
           <string>Previous Series</string>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="forwardSeries_button">
          <property name="toolTip">
           <string>Next Series</string>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="endSeries_button">
          <property name="toolTip">
           <string>Last Series</string>
          </property>
          <property name="text">
           <string>&gt;&gt;|</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="dataSelector_slider">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Scroll through the series</string>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLStereoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLStereoWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
